deepes/test/run_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+
+#----------------protobuf-------------#
+rm deepes.proto
```

Comment:
remove this line, as the following line will copy a new deepes.proto to the current directory.

Reply:
done